drift-integration-tests/src/test/java/com/facebook/drift/integration/TestClientsWithApacheServer.java

```diff
@@ -131,7 +130,7 @@ private static int testApacheServer(boolean secure, Transport transport, Protoco
         }
 
         try (TServerSocket serverSocket = createServerTransport(secure)) {
-            TServer server = new TSimpleServer(new Args(serverSocket)
+            TServer server = new TThreadPoolServer(new TThreadPoolServer.Args(serverSocket)
```

Comment:
wondering why this was needed ?

Reply:
This was changed to support multiple connections on server side.